docs/extensions/extension.md

```diff
@@ -9,7 +9,7 @@ But, some requirements don't match with those special resources or don't depend
 Gardener creates one `Extension` resource per registered extension type in `ControllerRegistration` per shoot. 
 
 ```yaml
-apiVersion: core.gardener.cloud/v1alpha1
+apiVersion: core.gardener.cloud/v1beta1
 kind: ControllerRegistration
 metadata:
   name: extension-example
```

Comment:
I guess the apiVersion of the Shoot example in this file could be also updated to `core.gardener.cloud/v1beta1`.